src/views/HomeView.vue

```diff
@@ -131,7 +139,7 @@ const getResult = async () => {
 .Container,
 .KnowledgeContainer {
   width: 800px;
-  height: auto;
+  height: 800px;
```

Comment:
All the sizes of the container should be variable. Width need to be measured in vw (view width) and height should be measured in vh (view height). 

100vw is 100% of the width of the user screen, and same for height